### PR TITLE
build: Update known-good files for 1.2.172 header

### PR DESCRIPTION
**Pull request by mikes-lunarg**
Changes:
- Update `known_good.json`

### CODE REVIEW

**Comment by mikes-lunarg** [comment]
Passed internal CI

**Comment by mikes-lunarg** [comment]
Passed macOS build

**Review by jeremyk-lunarg — APPROVED** [review]
Builds on macOS.
One small nit, but otherwise LGTM.

File: scripts/known_good.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "sub_dir" : "Vulkan-Headers",
       "build_dir" : "Vulkan-Headers/build",
       "install_dir" : "Vulkan-Headers/build/install",
-      "commit" : "4fe5828a0295a3c29a266e446ac6a6b51d768e48"
```
[the remark this answers is not in the document]
Comment (mikes-lunarg) [reply scripts/known_good.json:9]: Hmm good question, I think my editor is prettifying the json file on save. I'll put that space back in just to be consistent.

**Pull request merged**